internal/pkg/service/stream/components.go

```diff
@@ -22,6 +23,7 @@ const (
 	ComponentStorageCoordinator Component = "storage-coordinator"
 	ComponentStorageWriter      Component = "storage-writer"
 	ComponentStorageReader      Component = "storage-reader"
+	ComponentMigrator           Component = "migrator"
```

Comment:
We do not want migration component but CLI command. Are you able to drop this after migration easily? You need to write up commits which will be dropped after successful migration.
That's why having a cmd is easier, just drop the `migration` folder and done.

Reply:
Yeah it's easy to revert.